CT-1572 move libreoffice build

misaka opened the pull request.

aliuk2012 commented on spec/features/02_create/case_create_spec.rb:
```diff
@@ -61,9 +62,9 @@
 
     select drafter.email, from: 'assignment[assignee_id]'
-    click_button 'Create and assign case'
+    click_button 'Assign case'
```
Couldn't we use site prism?

EdwardAndress commented:
@aliuk2012 - yes, you're right about the policy.

EdwardAndress commented:
@aliuk2012 - I have, I think, addressed your comments regarding the policy :)

misaka committed:
CT-1572 add Dockerfile.base to build base image

This base Docker image is meant to speed up our app builds, since the base image
needs to containe LibreOffice which takes a good bit of time to install with
apt.

misaka committed:
CT-1572 change FROM image for docker build to our project's base

This base image will already have libreoffice installed, so this should speed up
build times and, if we're lucky, maybe even deploy times too.
